### DISCUSSION TITLE
What should be the vocabulary size for a blank model?

### DISCUSSION THREAD

**Discussion by sonynavdeep81** [explosion/spaCy#10022] [Help: Coding & Implementations] [answered]
I am loading a blank model. When I am checking the vocabulary length, it shows a non-zero value. Why is it so? Shouldn't it display 0?

Here is the code and the output:

![Screenshot (2)](https://user-images.githubusercontent.com/38432505/148903370-ac62e447-d3b4-412e-8a84-bd0709a81ad9.jpg)

**Comment by polm** [comment, marked as answer]
Blank Vocabs are initialized with some strings, like labels used for dependency parsing.

The length of a Vocab (or the strings in it) is basically an implementation detail, and shouldn't be relied on to count the words a model has seen or something.

**Reply by sonynavdeep81** [reply]
Thanks for the explanation.
